wagtail/admin/tests/pages/test_explorer_view.py

```diff
@@ -687,7 +687,7 @@ def test_nonadmin_sees_breadcrumbs_up_to_cca(self):
         """
         self.assertContains(response, expected, html=True)
         expected = """
-            <li class="w-h-full w-flex w-items-center w-overflow-hidden w-transition w-duration-300 w-whitespace-nowrap w-flex-shrink-0 w-font-bold w-max-w-0" data-breadcrumb-item hidden>
+            <li class="w-h-full w-flex w-items-center w-overflow-hidden w-transition w-duration-300 w-whitespace-nowrap w-flex-shrink-0 w-font-bold" data-breadcrumb-item hidden>
```

Comment:
```suggestion
            <li class="w-h-full w-flex w-items-center w-overflow-hidden w-transition w-duration-300 w-whitespace-nowrap w-flex-shrink-0 w-font-bold w-max-w-0" data-breadcrumb-item hidden>
```

Reply:
I did it because we are passing w-max-w-0 only to root element. This test works fine.

Reply:
Though why we use it only on root is a question

Reply:
If you look at the current main branch we use it in a few places. 

https://github.com/wagtail/wagtail/blob/911f17bf8fcf8ae6cbf747b0317e8cc596153197/wagtail/admin/templates/wagtailadmin/shared/breadcrumb-next.html#L28

https://github.com/wagtail/wagtail/blob/911f17bf8fcf8ae6cbf747b0317e8cc596153197/wagtail/admin/templates/wagtailadmin/shared/breadcrumb-next.html#L36

https://github.com/wagtail/wagtail/blob/911f17bf8fcf8ae6cbf747b0317e8cc596153197/wagtail/admin/templates/wagtailadmin/shared/breadcrumb-next.html#L53

The only breadcrumb item that does not use it is the trailing one. Which kind of makes sense as when it's used it does not need to expand.

Reply:
This is why I think the unit tests should be unchanged (for the explorer breadcrumbs and our code changes to support the expanded variant should simply not add the w-max-w-0

Reply:
When I say the tests should not change I mean they should be reverted to what is currently in the main branch.

Reply:
that's easier